CoreScriptsRoot/Modules/BackpackScript.lua

```diff
@@ -36,18 +36,19 @@ local ARROW_IMAGE_CLOSE = 'rbxasset://textures/ui/Backpack_Close.png'
 local ARROW_SIZE = UDim2.new(0, 14, 0, 9)
 local ARROW_HOTKEY = Enum.KeyCode.Backquote.Value --TODO: Hookup '~' too?
 local ARROW_HOTKEY_STRING = '`'
+local ARROW_HOVER_COLOR = Color3.new(0,162/255,1)
 
 local HOTBAR_SLOTS_FULL = 10
+local HOTBAR_SLOTS_VR = 6
```

Comment:
should we rename the other SLOTS to be more descriptive? full/vr/mini seems like a weird split